raiwidgets/raiwidgets/responsibleai_dashboard.py

```diff
@@ -30,7 +30,7 @@ class ResponsibleAIDashboard(Dashboard):
     """
     def __init__(self, analysis: RAIInsights,
```

Comment:
can you add some tests for this parameter in the python API?

Reply:
done